airdrop/getter.gno

```diff
@@ -0,0 +1,220 @@
+package airdrop
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/demo/uint256"
+)
+
+// GetAirdropConfig returns the current configuration of the airdrop as a JSON string.
+//
+// Example:
+//
+//	{
+//	  "refundable_timestamp": "1234567890",
+//	  "refund_to": "g1234567890abcdefghijklmnop"
+//	}
+func GetAirdropConfig(jsonStr string) string {
+	airdrop := createAirdropFromJSON(jsonStr)
+
+	node := json.ObjectNode("", nil)
+
+	timestamp := strconv.FormatUint(airdrop.config.RefundableTimestamp, 10)
+	err := node.AppendObject("refundable_timestamp", json.StringNode("refundable_timestamp", timestamp))
```

Comment:
If it's `timestamp` value, shouldn't it be NumberNode?? Not StringNode

Reply:
In the p/json. `NumberNode` was specified as `float64`, but because the value is sometimes stored in scientific notation instead of a timestamp, I specified it as `StringNode`.

Reply:
What do you mean by `value` from `sometimes stored in scientific notation instead of a timestamp` ??

Reply:
For example, when I save a value like 1234567890, it sometimes came back as 1.23456789e9. not always but just in case.